Misc/NEWS.d/next/Build/2024-08-01-10-55-15.gh-issue-122573.4-UCFY.rst

```diff
@@ -0,0 +1,2 @@
+The Windows build of CPython now requires 3.10 as a minimum version, because
+of the use of match case statements within the code of Python itself.
```

Comment:
```suggestion
The Windows build of CPython now requires 3.10 or newer.
```

Moreover, I suggest moving this to the _Windows_ NEWS category.

Reply:
It's merged now, but I'd have suggested keeping it in Build as it doesn't have any impact on users of a pre-built Python (who, on Windows, far exceed the number of people who build).